shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -70,6 +72,32 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent*)event {
   return NO;
 }
 
+- (void)applyBackdropFilter {
+    if(_backdropFilterApplied) // TODO EMILY: This check is not needed, but it might improve space/time performance.
```

Comment:
nits: 
if(_backdropFilterApplied) {
  return;
}

Reply:
The "todo" message seems unnecessary. I think it is needed right?

Reply:
This is a good idea to not applying the blur filter again and again, but we need to be care to not escape if the inputRadius is different. So instead of a boolean, maybe we store the CAFilter as a property and check if the new blur intensity is going to be the same as the CAFilter?